app/assets/sass/patterns/_check-your-answers.scss

```diff
@@ -1,14 +1,75 @@
+
+// please adjust, depending on length of questions and answers and number of sections
+$cya-first-column-width: 30% !default;
+// recommended for single sections: auto
+// recommended for longer questions: 50%
```

Comment:
The default wouldn't change when services override it, it would still say `30%`. But I can add a comment that the default is meant for short questions.

What else could we say instead of "single section"? Maybe "single set of questions"? Not sure... Any ideas?